lib/ws_proxy.rb

```diff
@@ -27,8 +27,10 @@ def start_proxy
     begin
       cmd  = "#{ws_proxy} --daemon --idle-timeout=#{idle_timeout} --timeout=#{timeout} #{port} #{host}:#{host_port}"
       cmd += " --ssl-target" if ssl_target
-      cmd += " --cert #{Setting[:websockets_ssl_cert]}" if Setting[:websockets_ssl_cert]
-      cmd += " --key #{Setting[:websockets_ssl_key]}" if Setting[:websockets_ssl_key]
+      if Setting[:websockets_encrypt]
+        cmd += " --cert #{Setting[:websockets_ssl_cert]}" if Setting[:websockets_ssl_cert]
+        cmd += " --key #{Setting[:websockets_ssl_key]}" if Setting[:websockets_ssl_key]
```

Comment:
Maybe you want to raise an error if the cert or key are not set, but `websockets_encrypt` is true. Raising the error at `ws_proxy.rb` is probably too late as I can't see how to let the user know about this at this point, probably the controller should handle it instead.